### DISCUSSION TITLE
Adding custom tags to store volume and issue ID's

### DISCUSSION THREAD

**Discussion by ErenPhayte** [comictagger/comictagger#591] [Ideas]
Hi,

Is there a way to store [Issue ID 660010] sitting in notes into two custom fields? I have comicrack, and used a plugin for the data however it's not as accurate as this tool. However, all my comics have two fields

comicvine_issue
comicvine_volume

and it would be great if there was a way to populate these two fields with the respective data, especially when doing refreshes on data.

**Comment by Drudoo** [comment]
The issue with ComicRack's custom values (like `comicvine_issue` and `comicvine_volume`) is that they are not stored in the `ComicInfo.xml` file in the cbz archive. The values are stored in the `ComicDb.xml` database that ComicRack uses. Each entry in ComicRack has a `<CustomValueStore>` where these values are stored. 

So even if comictagger could store them in the `ComicInfo.xml` file, ComicRack wouldn't read them.

**Comment by lordwelch** [comment]
Custom metadata plugins can now be created. See https://github.com/comictagger/comicinfoxml as a reference.
Currently you must have comictagger installed via pip to use a custom plugin, there are plans to allow loading plugins from the config directory in the future.

**Comment by lordwelch** [comment]
Local plugins can now be used on the latest beta see the [wiki] for more info

[wiki]: https://github.com/comictagger/comictagger/wiki/Installing-plugins

**Comment by CountZeroOr** [comment]
Where can we find the list of current plugins?

**Reply by lordwelch** [reply]
There currently is no list. You can check the ComicTagger organisation and @mizaki for all the current ones

**Reply by mizaki** [reply]
I have updated https://github.com/comictagger/comictagger/wiki/MetadataSchemes with links to available plugins
